src/components/v2/shared/DistributionSplitsSection/DistributionSplitCard.tsx

```diff
@@ -0,0 +1,200 @@
+import { Button, Col, Row, Space } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { Split } from 'models/v2/splits'
+import { useContext, useState } from 'react'
+import { permyriadToPercent } from 'utils/formatNumber'
+import FormattedAddress from 'components/shared/FormattedAddress'
+import { formatDate } from 'utils/formatDate'
+import {
+  CrownFilled,
+  LockOutlined,
+  CloseCircleOutlined,
+} from '@ant-design/icons'
+import { V2ProjectContext } from 'contexts/v2/projectContext'
+
+import DistributionSplitModal from './DistributionSplitModal'
+import { CurrencyName } from 'constants/currency'
+
+export default function DistributionSplitCard({
```

Comment:
The cards that appear in this list
<img width="690" alt="Screen Shot 2022-04-24 at 10 29 18 am" src="https://user-images.githubusercontent.com/96150256/164969818-f6b35f4c-1b5f-41b1-9aaf-7ae750bcf8cd.png">

Reply:
THANK YOU for screenshot <3